src/layer/layer_impl.rs

```diff
@@ -17,15 +17,39 @@ use crate::{
 
 use super::*;
 
-static SPAN_DATA: LazyLock<RwLock<HashMap<tracing::span::Id, SpanData, BuildHasherDefault<FNV1aHasher64>>>> = LazyLock::new(|| {
-    RwLock::new(HashMap::with_hasher(BuildHasherDefault::<FNV1aHasher64>::default()))
-});
+static SPAN_DATA: RwLock<HashMap<tracing::span::Id, SpanData, FNV1aHasher64HasherBuilder>> =
+    RwLock::new(HashMap::with_hasher(FNV1aHasher64HasherBuilder::new()));
+
+#[derive(Default)]
+struct FNV1aHasher64HasherBuilder {}
+impl FNV1aHasher64HasherBuilder {
+    const fn new() -> Self {
+        Self {}
+    }
+}
+
+impl BuildHasher for FNV1aHasher64HasherBuilder {
+    type Hasher = FNV1aHasher64;
+    fn build_hasher(&self) -> Self::Hasher {
+        FNV1aHasher64::default()
```

Comment:
Why the extra level of indirection? Why not just call new?

Reply:
There isn't any new for FNV1aHasher64. Only default.

https://github.com/tmmcguire/hashers/blob/a1d5b1d6154e2c742e670e40487ba51e7e547e0c/src/lib.rs#L431

Reply:
Oh, right. I got distracted by the (probably unnecessary) `derive(Default)` for Hasher (which probably doesn't even need a new).